### DISCUSSION TITLE
Tamaño en Figma y Responsive Design

### DISCUSSION THREAD

**Discussion by DaviiidSG** [Launch-X-Latam/MisionFrontEnd#126] [Q&A] [answered]
Buenas tardes, como ya sabemos, al hacer el boceto en Figma debemos elegir un tamaño de pantalla pero ese tamaño es único mientras que en el responsive design debemos tener diferentes diseños en diferentes tamaños de pantalla, mi duda es: en un proyecto real debemos hacer diferentes diseños en Figma para cada tamaño de pantalla?

**Comment by LaloValle** [comment, marked as answer]
Generalmente se realizan 2 o 3 diseños. Dependiendo del enfoque con el que desarrolles se realiza primero el diseño para una resolución de celular, le sigue (aunque es opcional) una pantalla de una tableta y finalmente un diseño para escritorio.
Lo que resta únicamente sería definir el rango de las resoluciones para acomodar el diseño de uno u otro, me parece que hay algunos estándares típicos que se utilizan pero sinceramente no los recuerdo :( .
